src/js/controller/model.js

```diff
@@ -223,12 +228,20 @@ define([
                 this.set('flashBlocked', false);
             }
 
+            // If there is no current provider, then the video tag might have a preset playback rate...
+            if (!_provider) {
+                this.set('defaultPlaybackRate', _currentProvider.getPlaybackRate());
+            }
```

Comment:
Changed so that we set `playbackRate` on the model based on the tag's rate.  `defaultPlaybackRate` is specifically based on user selection/API selection.